src/main/java/bio/terra/cda/app/controller/MetaApiController.java

```diff
@@ -30,16 +36,27 @@ public MetaApiController(ApplicationConfiguration applicationConfiguration) {
 
   @Override
   public ResponseEntity<SystemStatus> serviceStatus() {
-    HttpStatus httpStatus = HttpStatus.OK;
-
-    SystemStatusSystems otherSystemStatus =
-        new SystemStatusSystems().ok(true).addMessagesItem("everything is fine");
+    SystemStatus systemStatus = new SystemStatus();
+    try {
+      BigQuery bigQuery = BigQueryOptions.newBuilder().setProjectId("gdc-bq-sample").build().getService();
```

Comment:
Duplicating the code in `QueryService` here can result in an incorrect test. Instead this code should call a method in `QueryService` that returns the BQ status, by using the bigquery object in that class.